### PR TITLE
Bug 1053747 - Add a collapsible rocketbar to homescreens.

### PR DESCRIPTION
**Pull request by Cwiiis**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1976e9b076a2828bbea8ebc9f0db26a414a5dd19

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0a4ea965e0b8af2437175f5a4bfd0c7641e6efb5

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=91e47779bab281f43ff5c47b40a4fef27252d88e

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8bdbbd6ab58142779a0574f594d9cc7908210f2b

File: apps/system/js/app_chrome.js [hunk 1]
```diff
@@ -80,6 +82,13 @@
       this.app.element.classList.remove('search-app');
     }
 
+    if (this.app.isHomescreen) {
+      this.app.element.classList.add('home-app');
```
Comment (alivedise) [inline apps/system/js/app_chrome.js]: We don't need this...you could use .homescreenWindow in CSS if you like

File: apps/system/js/app_chrome.js [hunk 2]
```diff
@@ -80,6 +82,13 @@
...
+      this.title.setAttribute('data-l10n-id', 'search-or-enter-address');
+    } else {
+      this.app.element.classList.remove('home-app');
```
Comment (alivedise) [inline apps/system/js/app_chrome.js]: the same, we don't need it

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4bc4949375230625cab203124b115c34567c91a4

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=88f7f04b3a4b8eb26afa3601cd9f2ed23acbc2d2

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=95765ae7605cfefc4894cd395fa325202279daf4

**Pull request merged**